script/deb-s3-upload

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+_dirs=$(find "$*" -maxdepth 2 -mindepth 2 -type d -exec realpath --relative-to="$*" {} \;)
+for _dir in $_dirs; do
+    echo "Uploading deb packages in $_dir"
+    tree "$*/$_dir"
```

Comment:
same applies here, I guess. All good if it's deliberate.

Reply:
Yeah, the debug statement was deliberate. But I [removed this one](https://github.com/pgxman/buildkit/pull/16/commits/dc72b91d0d4b98135cedaf5cbe3045bbe6af4690) considering we have the one for the whole dir above